Sources/Build/PkgConfig.swift

```diff
@@ -0,0 +1,161 @@
+        }
+        
+        for dep in parser.dependencies {
+            var pkg = try PkgConfig(name: dep)
+            try pkg.load()
+            self.cFlags += pkg.cFlags
+            self.libs += pkg.libs
+        }
+    }
+}
+
+private struct PkgConfigParser {
+    let pcFile: String
+    var variables = [String: String]()
+    var dependencies = [String]()
+    var cFlags: String?
+    var libs: String?
+    
+    enum Token {
+        case CFlats(String)
+        case Libs(String)
+        case Dependencies(String)
+        case Variable(name: String, value: String)
+    }
+    
+    init(pcFile: String) {
+        self.pcFile = pcFile
+    }
+    
+    mutating func parse() throws {
+        let file = File(path: self.pcFile)
+        for line in try file.enumerate() {
+            if !line.characters.contains(":") && line.characters.contains("=")  {
+                let equalsIndex = line.characters.index(of: "=")!
+                let name = line[line.startIndex..<equalsIndex]
+                let value = line[equalsIndex.successor()..<line.endIndex]
+                variables[name] = resolveVariables(value)
+            } else if line.hasPrefix("Requires: ") {
+                dependencies = parseDependencies(value(line: line))
+            } else if line.hasPrefix("Libs: ") {
+                libs = resolveVariables(value(line: line)).chomp()
+            } else if line.hasPrefix("Cflags: ") {
+                cFlags = resolveVariables( value(line: line)).chomp()
+            }
+        }
+    }
+    
+    func parseDependencies(_ depString: String) -> [String] {
+        let exploded = depString.characters.split(separator: " ").map(String.init)
+        let operators = ["=", "<", ">", "<=", ">="]
+        var deps = [String]()
+        var skipNext = false
+        for depString in exploded {
+            if skipNext {
+                skipNext = false
+                continue
+            }
+            if operators.contains(depString) {
+                skipNext = true
+            } else {
+                deps.append(depString)
+            }
+        }
+        return deps
+    }
+    
+    func resolveVariables(_ line: String) -> String {
+        func resolve(_ string: String) -> String {
+            var resolvedString = string
+            guard let dollar = resolvedString.characters.index(of: "$") else { return string }
+            guard let variableEndIndex = resolvedString.characters.index(of: "}") else {return string }
+            let variable = string[dollar.successor().successor()..<variableEndIndex]
+            let value = variables[variable]!
+            resolvedString = resolvedString[resolvedString.startIndex..<dollar] + value + resolvedString[variableEndIndex.successor()..<resolvedString.endIndex]
+            return resolvedString
+        }
+        var resolved = line
+        while resolved.characters.contains("$") {
+            resolved = resolve(resolved)
+        }
+        return resolved
+    }
```

Comment:
Again, need comments about what this is trying to do. Other notes:
 - This code should be more strict, it isn't validating the syntax (e.g., it accepts "$(foo}").
 - This should build up the result in pieces by parsing left to right, rather than repeatedly rescanning the string. Right now it is `O(n^2)` when it should be `O(n)`.